src/app/study-flashcard/study-flashcard.component.html

```diff
@@ -0,0 +1,10 @@
+<div class="back-to-view">
+  <h2>back-to-view </h2>
+</div>
+
+<div class="card-space">
+  <h2>Card-space</h2>
+  <div class="navigation-bar">
```

Comment:
Change the name of this to progress bar to avoid confusion between navbar and this part